app/models.py

```diff
@@ -65,5 +66,6 @@ def serialize(self):
                 "from": self.sentence.lang.language,
                 "to": self.targetlang.language},
             "sentence": self.sentence.text,
-            "audiofile": self.audiofile.name
+            "audiofile": self.audiofile.name,
```

Comment:
Nit : be consistent "recordedOn" vs "audiofile"

Reply:
Okay. 
I choose "audiofile" style.